lib/components/card.css

```diff
@@ -0,0 +1,83 @@
+.card-badge,
+.card-dismiss,
+.card-text-overlay,
+.card-text-only,
+.card-horizontal,
+.card-vertical,
+.card-shadow {
+  display: flex;
+  flex-direction: column;
+  width: 18rem;
```

Comment:
great using comma-separated for common properties

Reply:
Thanks!
Will update as suggested.